Sources/_CertificateInternals/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 ##===----------------------------------------------------------------------===##
 
 add_library(_CertificateInternals
-  "_TinyArray.swift")
+)
```

Comment:
Is this change intended? I get a cmake error now this is a library without any sources.

Reply:
Hrm, that looks incorrect but CI should have caught it.